### PR TITLE
hubble-relay: add an option to run pprof

### LINKED ISSUE #11460: hubble-relay: add pprof support
PR #11372 added support for gops. As gops can take advantage of pprof for profiling, it would be nice to add pprof support as well to ease in debugging/troubleshooting issues.

### PR DESCRIPTION
**Pull request by rolinh**
Closes: #11460

### CODE REVIEW

**Review by gandro — APPROVED** [review]

File: hubble-relay/cmd/serve/serve.go [hunk 1]
```diff
@@ -51,6 +53,9 @@ func New() *cobra.Command {
 	cmd.Flags().BoolVarP(
 		&f.debug, "debug", "D", false, "Run in debug mode",
 	)
+	cmd.Flags().BoolVar(
```
Comment (sayboras) [inline hubble-relay/cmd/serve/serve.go:56]: What do think about supporting this flag in helm chart as well https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/charts/hubble-relay/templates/deployment.yaml#L39?

Comment (rolinh) [reply hubble-relay/cmd/serve/serve.go:56]: Given that there is already such an option for the cilium agent, I'd say that it'd be good to have for hubble-relay as well, yes.

**Review by michi-covalent — APPROVED** [review]

**Review by glibsm — APPROVED** [review]

**Comment by gandro** [comment]
test-me-please

**Comment by tgraf** [comment]
test-me-please

**Comment by Rolinh** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Comment by Rolinh** [comment]
retest-4.9

**Comment by Rolinh** [comment]
retest-runtime

**Comment by aanm** [comment]
hit https://github.com/cilium/cilium/issues/11512

**Pull request merged**